test/connector/tcp/mysql/docker-compose.yml

```diff
@@ -56,7 +56,7 @@ services:
       dockerfile: Dockerfile.dev
     command: sleep 999d
     environment:
-      TEST_ROOT: /secretless/test/mysql_handler
+      TEST_ROOT: /secretless/test/connector/tcp/mysql
```

Comment:
It's not a blocker but keep in mind that integration tests usually cover much more functionality than just the module they are testing so this mirrored nesting under `test` might be overkill